<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1820</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CARE_ROBOT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(59, 59, 59);</string>
  </property>
  <widget class="QLabel" name="camera_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>220</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="state_label">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>200</y>
     <width>771</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="state2avg_label">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>360</y>
     <width>771</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>510</y>
     <width>431</width>
     <height>190</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 40pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>시작</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chart_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>510</y>
     <width>341</width>
     <height>391</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 40pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>문진표</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>720</y>
     <width>431</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 40pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>초기화</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="loading_bar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>920</y>
     <width>1761</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 24pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>1251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman L</family>
     <pointsize>42</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 42pt &quot;URW Bookman L&quot;;</string>
   </property>
   <property name="text">
    <string>Health Care Robot</string>
   </property>
  </widget>
  <widget class="QLabel" name="reset_label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>200</y>
     <width>220</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
